index.html

```diff
@@ -183,12 +211,12 @@ <h2 class="contact-us">CONTACT US</h2>
         </ul>
     </form>
```

Comment:
- [ ] [OPTIONAL] because this button (`<button class="my-form-button" type="submit">See project</button>`), it's better to give it a text of meningful action like Submit, Get in touch or Send, kindly consider changing this